### PR TITLE
Initial Blueprint Code

### PR DESCRIPTION
**Pull request by blakeburch**
These scripts contain the basic functionality to:

1. Execute a sync on a specific connector (data source)
2. Check for the sync status of a specific connector. Fivetran only provides this information via two fields, `last_success` and `last_failure`, so we have to compare those timestamps to the execution time.
3. No1 & No2 combined (sit and wait).
4. Update a connector. Primary function is to set the schedule to manual (when using Shipyard as the primary scheduler), trigger a historical sync, or pause the connector.

Per our recent conversation, if the scripts are run on Shipyard `pickle` files will include log IDs and downstream runs will look for upstream log IDs in the pickle file name. In this way, the scripts can be run in parallel without `.pickle` overlap.

### CODE REVIEW

File: fivetran_blueprints/execute_sync.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import pickle
+import sys
+import requests.auth
+import datetime
+import pytz
+
+try:
+    import check_sync_status
+except BaseException:
+    from . import check_sync_status
```
Comment (ericelsken) [inline fivetran_blueprints/execute_sync.py:16]: What's the point of this? Is there not a deterministic way of importing this?

**Review by ericelsken — APPROVED** [review]
I know all of the code has been written, and it's all in Python. But is it worth it to change the variable storing in a pickle file to something that more than python can read? Or do so when the languages change or a user asks for it?

File: fivetran_blueprints/execute_sync.py (hunk 1 above)
Comment (blakeburch) [reply fivetran_blueprints/execute_sync.py:16]: @ericelsken I tested this back when we made the dbt_cloud blueprints. When importing a module, there's a difference when the scripts just live locally or when they've been imported via pip. This was the easiest way I found to account for both instances.

**Comment by blakeburch** [comment]
> I know all of the code has been written, and it's all in Python. But is it worth it to change the variable storing in a pickle file to something that more than python can read? Or do so when the languages change or a user asks for it?

@ericelsken That thought popped into my mind. Pickle was easiest since the purpose is to help link the Blueprints, which are all written in Python, together. Plus pickle is specifically designed for sharing stored variable/class data.

I think we can address it in the future if that need arises. I imagine JSON file storage would work best.

**Pull request merged**